includes/class-sensei-grading-main.php

```diff
@@ -288,7 +288,7 @@ protected function get_row_data( $item ) {
 				'action' => $grade_link,
 			), $item, $course_id );
 
-		return $column_data;
+		return wp_kses_post( $column_data );
```

Comment:
`$column_data` is not a string.

Reply:
Fixed in https://github.com/Automattic/sensei/pull/2296/commits/cfcf94c9d378385c29a535de7db4e243348c406e.